### PR TITLE
P2022 177/navigate to details screen

### PR DESCRIPTION
**Pull request by Lucas-Fresko**
Adds navigation to DetailsActivity from MainActivity using temporary image view.

Adds DetailsActivity to the manifest.

### CODE REVIEW

File: app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,24 @@
 package com.intive.patronage22.intivi
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
```
Comment (Bombozaur666) [inline app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt:6]: Do you still need this library?

Comment (Lucas-Fresko) [reply app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt:6]: It's not needed.

In the future, AS can be instructed to optimise imports on the fly or at the stage of a git commit.

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#optimize-imports

**Review by Bombozaur666 — APPROVED** [review]

File: app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+import android.view.View
+import android.widget.ImageView
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        clickNavigate(findViewById<ImageView>(R.id.imageView), DetailsActivity::class.java)
+
```
Comment (kosiara) [inline app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt:16]: remove needless empty line

**Review by kosiara — APPROVED** [review]
When empty line is removed - you can merge it

File: app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt (hunk 2 above)
Comment (Lucas-Fresko) [reply app/src/main/java/com/intive/patronage22/intivi/MainActivity.kt:16]: Which one, top, bottom, or both?

**Pull request merged**